lib/Dialect/CGGI/Transforms/BooleanVectorizer.cpp

```diff
@@ -205,9 +200,8 @@ bool tryBoolVectorizeBlock(Block *block, MLIRContext &context) {
   return madeReplacement;
 }
 
-struct BooleanLineVectorizer
-    : impl::BooleanLineVectorizerBase<BooleanLineVectorizer> {
-  using BooleanLineVectorizerBase::BooleanLineVectorizerBase;
+struct BooleanVectorizer : impl::BooleanVectorizerBase<BooleanVectorizer> {
```

Comment:
like Jeremy's comment (I think on the other PR?) you might want to comment why the vectorizer code isn't shared with the straight-line-vectorizer (e.g. it has a different compatible op, or that you could improve it beyond the straight-line-vectorizer)

Reply:
Wrote a small explanation in the description of the pass